internal/newusers/cache/db_test.go

```diff
@@ -585,17 +471,14 @@ func initCache(t *testing.T, dbFile string) (c *cache.Cache, cacheDir string) {
 		createDBFile(t, filepath.Join("testdata", dbFile+".db.yaml"), cacheDir)
 	}
 
-	expiration, err := time.Parse(time.DateOnly, "2004-01-01")
-	require.NoError(t, err, "Setup: could not parse time for testing")
-
-	c, err = cache.New(cacheDir, cache.WithExpirationDate(expiration))
+	c, err := cache.New(cacheDir)
 	require.NoError(t, err)
 	t.Cleanup(func() { c.Close() })
 
 	return c, cacheDir
 }
 
-func requireGetAssertions[E any](t *testing.T, got E, wantErrType, err error, c *cache.Cache, cacheDir string) {
+func requireGetAssertions[E any](t *testing.T, got E, wantErrType, err error, _ *cache.Cache, cacheDir string) {
```

Comment:
if we don’t need the cache object anymore, shouldn’t it just be stripped from the method signature?